exampleCourse/questions/examplesNumberInput/question.html

```diff
@@ -0,0 +1,81 @@
+<div class="card my-2">
+  <div class="card-body">
+    Questions below were designed to showcase the different features of 
+    <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a>. 
+    The goal of <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a>
+    is to allow users to enter <b>real or complex values</b> to be checked within a specific tolerance.
+  </div>
+
+  <div class="card-body">
+    <pl-question-panel>
+      <p>Within this question, you can observe the default behavior of the <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a> element.
+        In particular, the default input field has help text as a placeholder as well as a question button and is set to use <code>"relabs"</code> to check the tolerance, which is a combination of relative (<code>"rtol"</code>) and absolute (<code>"atol"</code>) tolerances.</p>
+      <p> Solve for the value of $c$ given $c^2 = {{params.a}}^2 + {{params.b}}^2$ </p>
+    </pl-question-panel>
+    <pl-number-input answers_name="c_1" />
+  </div>
+</div>
+
+<div class="card my-2">
+  <div class="card-body">
+    <pl-question-panel>
+      <p>This question modifies the default relative tolerance (of <code>1e-3</code>) and absolute tolerance (of <code>1e-8</code>) underneath <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a> element. Furthermore, we add a label to the front of the input field to specify what the field should receive. </p>
+      <p> Solve for the value of $c$ given $c^2 = {{params.a}}^2 + {{params.b}}^2$ </p>
+    </pl-question-panel>
+    <pl-number-input answers_name="c_2" label = "Enter the Value for $c$:" rtol = "1e-3" atol="1e-10"/>
```

Comment:
The more canonical label from the rest of examples would be `$c = $`

Reply:
I completely agree with you! I didn't ask Nicolas to modify this because I was not the one that created this sample question. If there is a consent here, I agree that we should change all questions to have the label `$c = $`